pkg/daemon/cleanup/cleanup.go

```diff
@@ -54,7 +54,7 @@ func Deconfigure(cfg *rest.Config, ctx context.Context, configMapNamespace strin
 		if err != nil {
 			return err
 		}
-		desiredVersion, present := node.Annotations[nodeconfig.DesiredVersionAnnotation]
+		desiredVersion, present := node.Annotations[metadata.DesiredVersionAnnotation]
```

Comment:
In the future, please separate a "rename" change like this into a different commit, it doesn't have any impact on functionality but creates noise for the real functionality changes.

Reply:
I've split it out